src/Libraries/DSOffice/Excel.cs

```diff
@@ -82,10 +84,6 @@ public static Application RegisterAndGetApp()
             if (excel == null)
             {
                 excel = new Microsoft.Office.Interop.Excel.Application();
-                if (excel == null)
-                {
-                    throw new Exception("Excel could not be opened.");
-                }
```

Comment:
This will still result in `excel.Visible = ShowOnStartup` below to throw a `NullObjectReferenceException`, is this done because we need `AppDomain.CurrentDomain.ProcessExit += OnProcessExit;` to be done even when `excel` is `null`?